### DISCUSSION TITLE
关于群聊消息的转发

### DISCUSSION THREAD

**Discussion by kaka1969** [wechaty/wechaty#2101] [General]
比如a发了一个群聊消息给我，我想直接转发给b，现在用msg.forward(room)这个接口不行，在api里面也没有看到相关的内容，请问这方面的功能应该如何实现呢

**Comment by huan** [comment]
> msg.forward(room)这个接口不行

Could you explain why the `msg.forward(room)` does not work? It should work for your case.

**Comment by kaka1969** [comment]
你好，我指的群聊消息是那种多选之后合并转发的群聊消息，应该是msg.Type.MESSAGE_TYPE_CHAT_HISTORY这种类型的，这种类型用msg.forward(room)是不能转发的
